### ISSUE TITLE
[FEA] Add error message (or warning) if replace node does not exist

### ISSUE THREAD

**Issue by miguelangel** [rapidsai/gQuant#32]
Hi,

I have just noticed that, if the node detailed in the replace variable does not exist, gQuant does not complain at all.

It has taken me a few minutes to figure out why my workflow was failing.

![image](https://user-images.githubusercontent.com/26169771/62979867-3f810c00-be25-11e9-9bb4-2560c89d238c.png)

It would be great if we might add an error message or at least a warning if the nodes are not present.

I would vote for raising an error. Warnings might be disabled, and difficult to trace.

Looking for your comments.

Regards,
Miguel

**Comment by avolkov1** [comment]
That's a good point. Should check and output a runtime warning if there is a task-id in replace that is not present.
```
    def build(self, replace=None):
        # in build method of TaskGraph
        replace = dict() if replace is None else replace
        task_ids = [task[TaskSpecSchema.task_id] for task in self]
        for rkey in replace.keys():
            if rkey not in task_ids:
                warnings.warn(
                   'Replace task-id {} not found in task-graph'.format(rkey),
                   RuntimeWarning)

```

**Comment by yidong72** [comment]
Yes. I am going to create a PR for this

**Comment by avolkov1** [comment]
Fixed by PR #37

**Comment by avolkov1** [comment]
Fixed by PR #37
